frontend/components/Pages/HelpAndSupportPage.tsx

```diff
@@ -54,53 +45,69 @@ export const HelpAndSupport = () => {
     getTag();
   }, [getAppVersion, isMounted]);
 
+  const helpItems: HelpItem[] = useMemo(
+    () =>
+      compact([
+        latestTag
+          ? {
+              label: 'Latest release notes',
+              href: `${GITHUB_API_RELEASES}/tag/v${latestTag}`,
+              isExternal: true,
+            }
+          : null,
+        {
+          label: 'Olas community Discord server',
+          href: SUPPORT_URL,
+          isExternal: true,
+        },
+        {
+          label: 'Frequently asked questions',
+          href: FAQ_URL,
+          isExternal: false,
+        },
+        {
+          label: 'Pearl Terms and Conditions',
+          href: TERMS_AND_CONDITIONS_URL,
+          isExternal: false,
+        },
+      ]),
+    [latestTag],
+  );
+
   return (
-    <Card
-      title={<SettingsTitle />}
-      bordered={false}
-      styles={{ body: { paddingTop: 0, paddingBottom: 0 } }}
-      extra={isUserLoggedIn ? <GoToMainPageButton /> : <GoToLoginPageButton />}
-    >
-      <CardSection $borderBottom $padding="16px 24px 24px" vertical>
-        <Title level={5} className="m-0 mb-16 text-base">
-          Frequently asked questions
-        </Title>
-        <a target="_blank" href={FAQ_URL} className="mb-8">
-          Read FAQ {UNICODE_SYMBOLS.EXTERNAL_LINK}
-        </a>
-        {latestTag && (
-          <a
-            href={`${GITHUB_API_RELEASES}/tag/v${latestTag}`}
-            target="_blank"
-            rel="noopener noreferrer"
-            className="mb-8"
+    <Flex className="help-and-settings-container">
```

Comment:
we normally use Styled component for such cases, instead of making custom classes. can we do this here? alternatively can do something similar to what Mohan did 
```
const cardStyles: CSSProperties = {
  width: 552,
  margin: '0 auto',
} as const;
```